### ISSUE TITLE
Blank screen after installing via cocoapods

### ISSUE THREAD

**Issue by seblau** [react-native-community/lottie-react-native#344]
**System**

- react-native: 0.51.0
- lottie-react-native: 2.3.2

**Podfile**
```
...
use_frameworks!
...
pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga/yoga.podspec'
pod 'React', :path => '../node_modules/react-native', :subspecs => ['BatchedBridge',]
pod 'lottie-ios', :path => '../node_modules/lottie-ios'
pod 'lottie-react-native', :path => '../node_modules/lottie-react-native'
...
```

**Behaviour**

No build errors. I am also able to successfully import `LottieView` in my project and run it. But the animation is not playing - blank canvas.
If I don't install Lottie via cocoapods, but rather link it and add `Lottie.framework` manually to `Embedded Binaries` its working.

**Comment by sirajulm** [comment]
I'm not sure if this issue is still valid.
I have `react-native` app converted to `Swift` and uses `pod` for dependencies.
All I had to add was add to Podfile
`pod 'lottie-react-native', :podspec => '../node_modules/lottie-react-native/lottie-react-native.podspec'` and do `pod install`.

In JS add `LottieView` and I sampled the `PinJump.json` from the lottie examples, which is rendering the animation appropriately.


react-native: ^0.58.5
lottie-react-native: ^2.5.11
